control_flow.ipynb

```diff
@@ -483,7 +483,7 @@
    },
    "outputs": [],
    "source": [
-    "%%ipytest\n",
+    "# %%ipytest\n",
```

Comment:
Why did you comment that line? Leftover from testing?

Reply:
Yes, I was trying a few quick things and did not want to run the tests. You can make a suggestion to remove it

Reply:
Here we go.
```suggestion
    "%%ipytest\n",
```